test/mesh27cube.ply

```diff
@@ -0,0 +1,235 @@
+ply
```

Comment:
this .ply file is the result of a test (I don't know which one), and does not belong in the repository.

It would be good to find out which test creates it and to change it such that it is created in tempdir instead.

Reply:
Removed all the .ply files.

Perhaps `*.ply` should be added to `.gitignore`, for convenience?